client/src/components/dlcHover.jsx

```diff
@@ -1,11 +1,80 @@
 import React from 'react';
-// import { DlcTable } from './styles/auxstyles.js';
+import moment from 'moment';
+import {
+  StyledReviewSum, StyledBody, StyledTag, StyledRow, StyledRelease, StyledTitle, StyledReview, Screenshot,
+  StyledScreenshots, HoverBox, StyledContent
+} from './styles/hoverStyles.js';
+
+const positives = ['Overwhelmingly Positive', 'Mostly Positive', 'Very Positive'];
+const negatives = ['Overwhelmingly Negative', 'Mostly Negative', 'Very Negative'];
+
+const Content = (props) => {
+  let color;
+  if (negatives.indexOf(props.data.reviewSummary) > -1) {
+    color = '#a34c25';
+  } else if (positives.indexOf(props.data.reviewSummary) > -1) {
+    color = '#66c0f4';
+  } else {
+    color = '#b9a06a';
+  }
+
+  return (
+    <HoverBox id='hoverBox'>
+      <StyledContent>
+        <div className='hoverTopArea' style={{ display: 'none' }}></div>
+        <StyledTitle id='hoverTitle'>
+          {props.data.title}
+        </StyledTitle>
+        <StyledRelease id='hoverRelease'>
+          <span>{`Released: ${moment(props.data.releaseDate).format('MMM D, Y')}`}</span>
+          <div></div>
```

Comment:
What's the purpose of this empty div? if it's for a linebreak, alternatively, `<br />` might work better.